tools/clang/test/DXC/dot4add_i8_u8_packed-types.hlsl

```diff
@@ -0,0 +1,21 @@
+// RUN: %dxc /Od /T cs_6_8 %s | FileCheck %s
```

Comment:
Od introduces a lot of other aspects to this. It's fine to test it, but I'd like to see a test without it as well as the only reason it is required for this repro is because optimization passes remove the unused variables and calls. If you assign the results into a RWByteAddress buffer then the Od isn't needed:

```
RWByteAddressBuffer buf;
...
buf.Store<int>(0, c7);
```